lib/functions.sh

```diff
@@ -907,6 +907,41 @@ make_install_in() {
         logerr "------ Make install in $1 failed"
 }
 
+make_lintlibs() {
+    logmsg "Making lint libraries"
+
+    LINTLIB=$1
+    LINTLIBDIR=$2
+    LINTINCDIR=$3
+    LINTINCFILES=$4
+
+    [[ -z ${LINTLIB} ]] && logerr "not lint library specified"
+    [[ -z ${LINTINCFILES} ]] && LINTINCFILES="*.h"
+
+    cat <<EOF > ${DTMPDIR}/${PKGD}_llib-l${LINTLIB}
+/* LINTLIBRARY */
+/* PROTOLIB1 */
+#include <sys/types.h>
+#undef _LARGEFILE_SOURCE
+EOF
+    pushd ${DESTDIR}${LINTINCDIR} > /dev/null
+	sh -c "eval /usr/gnu/bin/ls -U ${LINTINCFILES}" | \
```

Comment:
I know we have a few other references to /usr/gnu/bin in the build infrastructure, but is there any reason to prefer this over simply `echo ${LINTINCFILES}`, given our /bin/sh is certainly capable of expanding the expressions?